extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisBackedBlockRDD.scala

```diff
@@ -70,12 +70,12 @@ class KinesisBackedBlockRDDPartition(
  */
 private[kinesis]
 class KinesisBackedBlockRDD[T: ClassTag](
-    @transient sc: SparkContext,
+    sc: SparkContext,
```

Comment:
This matches the change made to https://github.com/apache/spark/commit/c1bc4f439f54625c01a585691e5293cd9961eb0c#diff-3fdb38876be7c77f4568be49edba4c02L38